frontend/src/components/filters.svelte

```diff
@@ -1,8 +1,7 @@
 <script lang="ts">
-  export let movieChecked = true
-  export let tvChecked = true
-  export let search
+  import { filters } from "../stores/filters.js"
 
+  export let search
```

Comment:
```suggestion
  export let search: string
```

Reply:
Typed as a function `() => void`

Reply:
Nvm it was a func, but we talked about it on Discord :D